ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -440,7 +457,32 @@ objects:
         EOF
 
           openshift-install --dir=/tmp/artifacts/installer create manifests
-          rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-*.yaml /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machinesets-*.yaml
+          rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-*.yaml /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+        elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
+            cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1
+        baseDomain: ${BASE_DOMAIN}
```

Comment:
I think we should do https://github.com/openshift/release/blob/96079892ec5d381896ae569823b53c03df4fc24a/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L867 instead of all new ENVs

Reply:
BASE_DOMAIN is used quite a few places in this file, I was looking to see if I could simplify it without using the default on every use.

Reply:
Trying again with :=

Reply:
people use `mkpj` style flows to test the CI template locally... having to set different them for each platform is tough..

Reply:
Using `${BASE_DOMAIN:=origin-ci-int-aws.dev.rhcloud.com}` is not overriding the variable as expected, so we need another way to work around this.

Reply:
Will refactor on #5151 once it lands.

Reply:
Cleaned up with ${base_domain} in line with #5151